source/Makefile

```diff
@@ -87,7 +87,7 @@ OPTIMIZE :=
 # 	will recreate only the "locales/en.catkeys" file. Use it as a template
 # 	for creating catkeys for other languages. All localization files must be
 # 	placed in the "locales" subdirectory.
-LOCALES = 
+LOCALES = en de es fr ja
```

Comment:
You should not add any languages there until the application has actually been translated. People translating it will provide catkeys files for the other languages, and once we have those, we can add them here (and in the git repository).